File: csharp/checkout/GetStateProvincesForCountry.cs
using System;
using System.Collections.Generic;
using com.ultracart.admin.v2.Api;
using com.ultracart.admin.v2.Model;
using Newtonsoft.Json;

namespace SdkSample.checkout
{
    public class GetStateProvincesForCountry
    {
        /// <summary>
        /// A simple method for populating the state_region list boxes with all the states/regions allowed for a country code.
        /// </summary>
        public static void Execute()
        {
            // Reference Implementation: https://github.com/UltraCart/responsive_checkout
            // A simple method for populating the state_region list boxes with all the states/regions allowed for a country code.

            CheckoutApi checkoutApi = new CheckoutApi(Constants.ApiKey);

            String countryCode = "US";

            CheckoutStateProvinceResponse apiResponse = checkoutApi.GetStateProvincesForCountry(countryCode);
            List<StateProvince> provinces = apiResponse.StateProvinces;

            foreach (StateProvince province in provinces)
            {
                Console.WriteLine(JsonConvert.SerializeObject(province,
                    new JsonSerializerSettings { Formatting = Formatting.Indented }));
            }
        }
    }
}

File: java/src/checkout/GetStateProvincesForCountry.java
package checkout;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;
import com.ultracart.admin.v2.CheckoutApi;
import com.ultracart.admin.v2.models.CheckoutStateProvinceResponse;
import com.ultracart.admin.v2.models.StateProvince;
import com.ultracart.admin.v2.util.ApiException;
import common.Constants;

import java.util.List;

public class GetStateProvincesForCountry {
    /**
     * A simple method for populating the state_region list boxes with all the states/regions allowed for a country code.
     * Reference Implementation: https://github.com/UltraCart/responsive_checkout
     */
    public static void execute() throws ApiException {
        CheckoutApi checkoutApi = new CheckoutApi(Constants.API_KEY);

        String countryCode = "US";

        CheckoutStateProvinceResponse apiResponse = checkoutApi.getStateProvincesForCountry(countryCode);
        List<StateProvince> provinces = apiResponse.getStateProvinces();

        Gson gson = new GsonBuilder().setPrettyPrinting().create();
        for (StateProvince province : provinces) {
            System.out.println(gson.toJson(province));
        }
    }
}